src/components/templates/descopeMiddleware.tsx

```diff
@@ -116,12 +117,21 @@ export const DescopeMiddleware = ({ children }: { children: ReactNode }) => {
 					return;
 				}
 				clearLogs();
-
-				//const userOrganization = useOrganizationStore.getState().get(user.defaultOrganizationId);
-				//setCurrentOrganization(userOrganization);
-
 				gTagEvent(googleTagManagerEvents.login, { method: "descope", ...user });
 				setIdentity(user.email);
+
+				const { data: userOrganization, error: errorOrganization } = await OrganizationsService.get(
+					user.defaultOrganizationId
+				);
+				if (errorOrganization || !userOrganization) {
+					addToast({
+						message: (errorOrganization as Error).message,
+						type: "error",
+					});
+
+					return;
+				}
+				setCurrentOrganization(userOrganization);
```

Comment:
If `errorOrganization` is `undefined` and `userOrganization` is `undefined`, this will cause a crash because `(errorOrganization as Error).message` doesn't exist. We need to check them separately and display "Login failed, please try again later" if we have an `errorOrganization` or `!userOrganization`.

Also, we shouldn't display the error that comes from the backend in the toast, the user should understand what happened, and the error should be short and simple.